tests/bwc/test_upgrade.py

```diff
@@ -307,24 +313,25 @@ def assert_data_persistence(self,
 
             # to trigger `alter` stmt bug(https://github.com/crate/crate/pull/17178) that falsely updated the table's
             # version created setting that resulted in oids instead of column names in resultsets
-            cursor.execute('ALTER TABLE doc.parted SET ("refresh_interval" = 900)')
-
-            # older versions had a bug that caused this to fail
-            # Test that partition and dynamic columns can be created
-            key = "t_" + version.replace('.', '_')
-            obj = {key: True}
-            args = (str(uuid4()), version, obj)
-            cursor.execute(
-                'INSERT INTO doc.parted (id, version, cols) VALUES (?, ?, ?)',
-                args
-            )
-            cursor.execute('REFRESH TABLE doc.parted')
-            accumulated_dynamic_column_names.append(key)
-            cursor.execute('SELECT cols FROM doc.parted')
+            cursor.execute('ALTER TABLE doc.dynamic SET ("refresh_interval" = 900)')
+            cursor.execute('INSERT INTO doc.dynamic (o) values (?)', [{version: True}])
```

Comment:
don't we need to replace `.` with `_` here, since they create nested sub-cols?

Reply:
Thanks, it is already done [here](https://github.com/crate/crate-qa/pull/346/files#diff-da22f73840f87231551db9d0b0f9c4592e94b91a66208f3a45935705d162a617R291).